src/dct/Theater.lua

```diff
@@ -181,6 +181,7 @@ function Theater:export(_)
 		["theater"]  = env.mission.theatre,
 		["sortie"]   = env.getValueDictByKey(env.mission.sortie),
 		["assetmgr"] = self:getAssetMgr():marshal(),
+		["currentDate"] = os.date("*t")
```

Comment:
```suggestion
		["startdate"] = self.startdate
```
I think we want something like this. The current implementation will just save the latest date & time of the server. And since we update the state file every 7 minutes this field will just get updated and the "date" field already serves this purpose. The reason issue #40 wanted to date & time saved was so we could determine when the state file was first created. What is missing with what I propose is the reading in of the state file to save an attribute to theater and in `__init()` something like `self.startdate = os.date("*t")`